src/cardInfo.js

```diff
@@ -4,6 +4,8 @@ const NAME = 0
 const NORMAL_DESCRIPTION = 1
 const REVERSE_DESCRIPTION = 2
```

Comment:
are you still using these constants? Looks like the code that uses them is commented out